collect_app/src/main/res/values-de/strings.xml

```diff
@@ -338,9 +338,6 @@
   <string name="google_maps_sdk">Google Maps SDK</string>
   <string name="openstreetmap_sdk">OpenStreetMap SDK</string>
   <string name="show_map_sdk">Map-SDK-Optionen anzeigen</string>
-  <string name="openstreetmap">OpenStreetMap</string>
```

Comment:
Don't worry about translations, only the strings.xml file matters. The others will get pulled from Transifex (https://www.transifex.com/opendatakit/odk-collect/) and overwritten.